app/controllers/admin/articles_controller.rb

```diff
@@ -45,6 +45,8 @@ def update
       article.boosted_additional_articles = article_params[:boosted_additional_articles].to_s == "true"
       article.boosted_dev_digest_email = article_params[:boosted_dev_digest_email].to_s == "true"
       article.user_id = article_params[:user_id].to_i
+      article.second_user_id = article_params[:second_user_id].to_i
+      article.third_user_id = article_params[:third_user_id].to_i
```

Comment:
I don't think this is in the scope of _this_ PR, but seeing these changes here seems like a indicator that we probably want to move away from having explicit columns for each co-author id (`second_user_id`, `third_user_id`, etc.) and opt for a more flexible solution: `co_author_ids`. It would be nice for the controller to send along params like this:
```ruby
params: { co_author_ids: [2, 3] }
```
so that we can ask the article for its coauthors like so:
```ruby
article.co_author_ids
=> [2, 3]
```

which is more flexible than adding a column each time we want to add a co-author id :) it would also mean that we don't need to send along 2 params for effectively the same concept like we are doing in this PR.

You can imagine that the moment we want to allow authors or admins to add a _fourth_ co-author, this is going to be painful to do the way that we currently have it set up.

Reply:
This is a great suggestion @vaidehijoshi , but definitely a bit out of scope for this feature since the need for it is pretty high. I've created an issue for this on my team's cool-down project board so that we can circle back around to it, though. 😊

Reply:
I really like this refactor idea

Reply:
> I've created an issue for this on my team's cool-down project board so that we can circle back around to it

@juliannatetreault Yep, I think that's the right call; again, not a blocker, as I mentioned in my comment, but worth refactoring in the near term since we're already in the code.